.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
```

Comment:
As this actually releases/publishes a build, perhaps name it "Publish" or something?